lib/util/mp4_parser.js

```diff
@@ -111,6 +111,12 @@ shaka.util.Mp4Parser = class {
   parseNext(absStart, reader, partialOkay, stopOnPartial) {
     const start = reader.getPosition();
 
+    // size(4 bytes) + type(4 bytes) = 8 bytes
+    if (stopOnPartial && start + 8 >= reader.getLength()) {
```

Comment:
I think == should be fine, right?  If a box had only the required size & name fields, `start + 8 == reader.getLength()` should be acceptable.  Do you agree?  Should this become `>` instead of `>=`?

Also, it looks like a similar check is needed below, when we handle `FULL_BOX` types and read `versionAndFlags`.

Reply:
Yes, I agree `start + 8 == reader.getLength()` should be acceptable.